### PR TITLE
fix(compliance): print a user friendly message instead of a generic error message

### LINKED ISSUE #2587: jx compliance results command error due to no prior run
### Summary 

```jx compliance results``` command error due to no prior ```run```

Similar to #2464 but related to ```results``` rather than ```status```

### Steps to reproduce the behavior

```$ jx compliance results```

### Jx version

The output of `jx version` is:

```
NAME               VERSION
jx                 1.3.688
Kubernetes cluster v1.10.9-gke.5
kubectl            v1.13.1
helm client        v2.12.1+g02a47c7
helm server        v2.11.0+g2e55dbe
git                git version 2.20.1
Operating System   Mac OS X 10.14.2 build 18C54
```

### Kubernetes cluster

What kind of Kubernetes cluster are you using & how did you create it?

Kubernetes cluster v1.10.9-gke.5

### Operating system / Environment

Operating System   Mac OS X 10.14.2 build 18C54

### Expected behavior

```
No compliance results found. Use jx compliance run command to start the compliance tests.
You can watch the logs with jx compliance logs -f command.
```

### Actual behavior

```
error: failed to retrieve the compliance status: sonobuoy namespace does not exist: namespaces "heptio-sonobuoy" not found
```

### PR DESCRIPTION
**Pull request by ccojocar**
#### Submitter checklist

- [x] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Which issue this PR fixes

fixes #2587

* Requesting a pull to jenkins-x:master from ccojocar:fix-compliance-results-err
*
* Write a message for this pull request. The first block
* of text is the title and the rest is the description.

### CODE REVIEW

File: pkg/jx/cmd/compliance_results.go [hunk 1]
```diff
@@ -117,7 +117,8 @@ func (o *ComplianceResultsOptions) Run() error {
 
 	err = eg.Wait()
 	if err != nil {
-		return errors.Wrap(err, "failed to retrieve the results")
+		log.Infof("No compliance status found. Use %s command to start the compliance tests.\n", util.ColorInfo("jx compliance run"))
```
Comment (msigler-cb) [inline pkg/jx/cmd/compliance_results.go]: ```log.Infof("No compliance results found.``` rather than ```status```

Comment (ccojocar) [reply pkg/jx/cmd/compliance_results.go]: fixed

**Pull request merged**